.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -42,6 +42,14 @@ you can strike them out. They are available if for some reason the internal sgid
 - [ ] SGID10
 - [ ] ArcGIS Online
 
+<!-- 
+the data should be in good shape.
+-->
+# The data is high quality
+
+- [ ] Sweeper checks have run and passed
+- [ ] The minimum metadata fields are populated
```

Comment:
```suggestion
```
[Sweeper already checks metadata](https://github.com/agrc/sweeper/blob/master/src/sweeper/sweepers/metadata.py). If it's missing something we should add it to sweeper.

Reply:
great addition.  i'll post the metadata minimum requirements doc on the webpage.  also, we should also add "Comply with domain rules" and I can clean up that domain doc that lays out  our plan and rules.

Reply:
@stdavis maybe we can change the metadata piece to remind the user of our minimum requirements.